department-of-veterans-affairs/caseflow
# Make POA last synced at available to appeals

Resolves [CASEFLOW-957](https://vajira.max.gov/browse/CASEFLOW-957)

### Description
Exposes the BGS power of attorney last synced at value via the appeals controller for use on the case details page (coming in another PR).

### Acceptance Criteria

### Testing Plan
1. add a `binding.pry` to the [appeals controller power_of_attorney method](https://github.com/department-of-veterans-affairs/caseflow/blob/bgantick/CASEFLOW-957-poa-last-synced-backend/app/controllers/appeals_controller.rb#L60)
2. Load up both an AMA and legacy case details page and ensure you can get `appeal.poa_last_synced_at`
